controllers/iochaos/fs/types.go

```diff
@@ -80,6 +83,7 @@ func (r *Reconciler) Apply(ctx context.Context, req ctrl.Request, chaos reconcil
 		r.Log.Error(err, "chaos is not IoChaos", "chaos", chaos)
 		return err
 	}
+	r.Event(iochaos, v1.EventTypeNormal, utils.EventChaosToInject, "")
```

Comment:
I think we should send event after the chaos action happened.

Reply:
Makes more sense. Will change them later.